### ISSUE TITLE
Release 2019.11.x forthcoming [On Hold]

### ISSUE THREAD

**Issue by zoracon** [EFForg/https-everywhere#18618]
Type: other

Release Notes (So far):
- #18417 EASE HTTP Once Exception
- #18565 Add Private network IPs to exclusion for HTTPSE
- #18153 Refactor code for the browser session storage
- #18263 Revert icons back to previous state
- #18452 Normlise disabled site hostname and migrate stored list
- #18602 hsts-prune

Alerting: @J0WI , @TotalCaesar659 , @Bisaloo , @pipboy96 , @ivysrono , @cschanaj

**Comment by cschanaj** [comment]
@zorazon the fix for #18589 should be merged before the next release since most of the wildcard rules is not working right now. It affects all browsers which support WASM by default.

**Comment by ivysrono** [comment]
#18495

**Comment by cschanaj** [comment]
#18602

**Comment by zoracon** [comment]
I really want to get https://github.com/EFForg/https-everywhere-lib-core/pull/1 In this release. So holding off until Monday. If it can't happen then, I'll plan for a patch release when it occurs after this one.

**Comment by zoracon** [comment]
Update: Was in the process of release but came across some strange issues with the translations files and having a few process problems. Will be retrying first thing in the morning. Thank you for your patience.

**Comment by pipboy96** [comment]
@zoracon Should this be closed?

**Comment by zoracon** [comment]
@pipboy96 , yes for some reason the merge didn't close this out
